data-prepper-plugins/kafka-plugins/src/main/java/org/opensearch/dataprepper/plugins/kafka/configuration/KafkaProducerProperties.java

```diff
@@ -6,7 +6,22 @@
 import java.util.List;
 
 public class KafkaProducerProperties {
-    private static final String DEFAULT_BYTE_CAPACITY = "50mb";
+    static final String DEFAULT_BYTE_CAPACITY = "50mb";
```

Comment:
Might it make more sense to use the Kafka client defaults here? With this change, we will have to keep our defaults in sync with Kafka's.

I do also see that this has an advantage of allowing us to stay consistent between versions even if Kafka changes. But, I'm not sure if that is an ideal gain. Thoughts?

Reply:
This is existing value that I am not changing. For the rest of the KafkaProducerProperties that I added, I chose the values that are Kafka defaults.